src/Winium.Desktop.Driver/CommandExecutors/GetElementAttributeExecutor.cs

```diff
@@ -58,6 +58,20 @@ private static object PrepareValueToSerialize(object obj)
                 return controlType.ProgrammaticName;
             }
 
+            // Fix issue error when parse runtimeId attribute of element(return Int32[] array) by convert them into string
```

Comment:
from @zelonght: Leave not fix info here, it's like new feature added and git history is enough.